tools/cdf_creator/swe_cdf_creator.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python
+"""
+swe_cdf_creator.py
+usage: <python> <swe_cdf_creator.py> <configuration.cfg>
+
+Creates list of ascending list from long-term SWE VIC produced data.
+Period of time is defined in configuration file. 
+Applies a moving window-esque method to collect 
+5 days of data at a time and then sorts that data. 
+February 29th data is removed before being sorted.
+February 28th (plus surround two days) is stored at February 29th. 
```

Comment:
could you put a little more documentation on what the output should be (format, kinds of values). I got a little lost in reading this script.